app/PimRequirements.php

```diff
@@ -64,7 +64,6 @@ public function getPimRequirements(): array
     }
 
     /**
-<<<<<<< HEAD
```

Comment:
Problem with github ?

Reply:
"<<<<<<< HEAD" was in the file itself
I just remove it
I probably comes from a bad merge